inginious/frontend/plugins/manual_scoring/pages/api/preview_content.py

```diff
@@ -0,0 +1,38 @@
+import web
+
+from inginious.frontend.pages.api._api_page import APIError
+from inginious.frontend.parsable_text import ParsableText
+from inginious.frontend.plugins.statistics.pages.api.user.user_api import UserApi
+from inginious.frontend.plugins.utils import get_mandatory_parameter
+
+
+def _transform_rst_content(content):
+    """ transform the rst content """
+    try:
+        _check_string(content)
+    except APIError as error:
+        error.send()
+
+    if not content:
+        content = "**There is not content yet**\n============================"
+    comment = ParsableText(content)
+    return comment.parse()
+
+
+def _check_string(content):
+    """ Check if the content is a string """
+    if not isinstance(content, str):
+        raise APIError(400, "The content isn't a string")
+
+
+class PreviewContent(UserApi):
```

Comment:
As bellow, I do not understand why this API is created as I see that it only transforms rst text, isn't it possible to just send it directly to the template when this is rendered?